entries/d/draftkings.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "DraftKings": {
+    "domain": "draftkings.com",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://help.draftkings.com/hc/en-us/articles/4409699580691",
+    "keywords": [
+      "other"
```

Comment:
Betting seems to be an appropriate category

Reply:
It's a mix of `betting` and `gaming`, I reckon.